src/site/xdoc/download.xml

```diff
@@ -540,32 +528,32 @@ is found <a href='https://www.apache.org/licenses/exports/'>here</a>.
     &lt;groupId&gt;org.apache.james&lt;/groupId&gt;
     &lt;artifactId&gt;apache-mailet-crypto&lt;/artifactId&gt;
-    &lt;version&gt;3.3.0&lt;/version&gt;
+    &lt;version&gt;3.4.0&lt;/version&gt;
   &lt;/dependency&gt;
       </code>
     </pre>
 
     <p>Direct download link:
-      <a href="https://www.apache.org/dyn/closer.lua/james/mailets/3.3.0/apache-mailet-crypto-3.3.0.jar">(Jar)</a>
-      [<a href="https://apache.org/dist/james/mailets/3.3.0/apache-mailet-crypto-3.3.0.jar.sha1">SHA-1</a>]
-      [<a href="https://apache.org/dist/james/mailets/3.3.0/apache-mailet-crypto-3.3.0.jar.asc">PGP</a>]
+      <a href="https://repo1.maven.org/maven2/org/apache/james/apache-mailet-crypto/3.4.0/apache-mailet-crypto-3.4.0.jar">(Jar)</a>
+      [<a href="https://repo1.maven.org/maven2/org/apache/james/apache-mailet-crypto/3.4.0/apache-mailet-crypto-3.4.0.jar.sha1">SHA-1</a>]
+      [<a href="https://repo1.maven.org/maven2/org/apache/james/apache-mailet-crypto/3.4.0/apache-mailet-crypto-3.4.0.jar.asc">PGP</a>]
     </p>
     
   </section>
 
   <section name="Apache James Protocols">
   
-    <p>Apache James Protocols 3.3.0 is the latest stable version.</p>
+    <p>Apache James Protocols 3.4.0 is the latest stable version.</p>
 
     <ul>
       You can get the James implementation of various protocols using maven:
-      <li><a href="https://mvnrepository.com/artifact/org.apache.james.protocols/protocols-api/3.3.0">API</a></li>
-      <li><a href="https://mvnrepository.com/artifact/org.apache.james.protocols/protocols-netty/3.3.0">NETTY</a></li>
-      <li><a href="https://mvnrepository.com/artifact/org.apache.james.protocols/protocols-smtp/3.3.0">SMTP</a></li>
-      <li><a href="https://mvnrepository.com/artifact/org.apache.james.protocols/protocols-lmtp/3.3.0">LMTP</a></li>
-      <li><a href="https://mvnrepository.com/artifact/org.apache.james.protocols/protocols-imap/3.3.0">IMAP</a></li>
-      <li><a href="https://mvnrepository.com/artifact/org.apache.james.protocols/protocols-pop3/3.3.0">POP3</a></li>
-      <li><a href="https://mvnrepository.com/artifact/org.apache.james.protocols/protocols-managesieve/3.3.0">MANAGESIEVE</a></li>
+      <li><a href="https://repo1.maven.org/maven2/org/apache/james/protocols/protocols-api/3.4.0">API</a></li>
+      <li><a href="https://repo1.maven.org/maven2/org/apache/james/protocols/protocols-netty/3.4.0">NETTY</a></li>
+      <li><a href="https://repo1.maven.org/maven2/org/apache/james/protocols/protocols-smtp/3.4.0">SMTP</a></li>
+      <li><a href="https://repo1.maven.org/maven2/org/apache/james/protocols/protocols-lmtp/3.4.0">LMTP</a></li>
+      <li><a href="https://repo1.maven.org/maven2/org/apache/james/protocols/protocols-imap/3.4.0">IMAP</a></li>
+      <li><a href="https://repo1.maven.org/maven2/org/apache/james/protocols/protocols-pop3/3.4.0">POP3</a></li>
+      <li><a href="https://repo1.maven.org/maven2/org/apache/james/protocols/protocols-managesieve/3.4.0">MANAGESIEVE</a></li>
```

Comment:
the files should be hosted by Apache, no?

Reply:
the source code not the compiled artifacts

Reply:
Not sure about that I remember moving back things to dist not that long ago

Reply:
btw It was on mvnrepository and now on maven.org, none of which belongs to Apache.

I propose to keep it that way unless we are asked to distribute it only by apache infrastructure.